mcfly.bash

```diff
@@ -19,8 +19,8 @@ fi
 # MCFLY_SESSION_ID is used by McFly internally to keep track of the commands from a particular terminal session.
 export MCFLY_SESSION_ID=$(dd if=/dev/urandom bs=256 count=1 2> /dev/null | env LC_ALL=C tr -dc 'a-zA-Z0-9' | head -c 24)
 
-# Find the binary
-MCFLY_PATH=${MCFLY_PATH:-$(which mcfly)}
+# Find the binary and apply default arguments
+MCFLY_PATH="${MCFLY_PATH:-$(which mcfly)} --history_format bash"
```

Comment:
5e3161d changes this to detect if `which mcfly` fails, so let's delete this default for now, since it defaults to bash anyway.